<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>849</height>
   </rect>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="documentTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>899</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddCircle"/>
    <addaction name="actionAddRectangle"/>
    <addaction name="actionAddTriangle"/>
    <addaction name="actionRemoveShape"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToggleClipping"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionLoad_Experiment"/>
    <addaction name="actionRun_Task"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuShape"/>
   <addaction name="menuView"/>
   <addaction name="menuRun"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="shapeToolBar">
   <property name="windowTitle">
    <string>Graphics</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddRectangle"/>
   <addaction name="actionAddCircle"/>
   <addaction name="actionAddTriangle"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Undo Stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QUndoView" name="undoView">
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAddRectangle">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Add Rectangle</string>
   </property>
  </action>
  <action name="actionAddCircle">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/circle.png</normaloff>:/icons/circle.png</iconset>
   </property>
   <property name="text">
    <string>Add Circle</string>
   </property>
  </action>
  <action name="actionRemoveShape">
   <property name="text">
    <string>Remove Shape</string>
   </property>
  </action>
  <action name="actionToggleClipping">
   <property name="text">
    <string>Toggle Clipping</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Begin Debugging</string>
   </property>
  </action>
  <action name="actionAddTriangle">
   <property name="icon">
    <iconset resource="workstation.qrc">
     <normaloff>:/icons/triangle.png</normaloff>:/icons/triangle.png</iconset>
   </property>
   <property name="text">
    <string>addTriangle</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad_Experiment">
   <property name="text">
    <string>Load Experiment</string>
   </property>
  </action>
  <action name="actionRun_Task">
   <property name="text">
    <string>Run Task</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUndoView</class>
   <extends>QListView</extends>
   <header>qundoview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="workstation.qrc"/>
 </resources>
 <connections/>
</ui>
